db/migrate/20230816073715_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :body
+      t.references :post, null: false, foreign_key: true
+      t.references :author, null: false, foreign_key: true
```

Comment:
Author and comments would not be having any associations. Please remove it as discussed.

Reply:
removed association